idf_build_apps/build_job.py

```diff
@@ -12,9 +15,18 @@
 )
 
 
-class BuildJob(BaseModel):
+class OverridePriority(Enum):
+    OVERRIDE_SDKCONFIG = 'override_sdkconfig'
+    OVERRIDE_SDKCONFIG_FILE = 'override_sdkconfig_file'
+
+
+class BuildAppJob(BaseModel):
     PARALLEL_INDEX_PLACEHOLDER: t.ClassVar[str] = '@p'  # replace it with the parallel index
 
+    override_sdkconfig_items: t.ClassVar[dict[str, t.Any]] = {}
+    override_sdkconfig_files_name: t.ClassVar[t.Tuple[str]] = ()
+    override_priority: OverridePriority = OverridePriority.OVERRIDE_SDKCONFIG
```

Comment:
we don't need these classvars actually.

The `override_sdkconfig_items` could be calculated in `pre_find_apps`

Reply:
Removed